### PR TITLE
IMicrosoftAuthService.getAccounts() and removeAccount()

### PR DESCRIPTION
**Pull request by heidijinxujia**
- Add the fix for acquireToken silent with broker
- Add the implementation for getAccounts on MSAL side
- Add the implementation for removeAccount on MSAL side
- Update the msal test app

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java [hunk 1]
```diff
@@ -525,22 +618,47 @@ public boolean removeAccount(@Nullable final IAccount account) {
...
                         account.getHomeAccountIdentifier().getIdentifier(),
                         realm
                 ).isEmpty();
+
+        if (MSALControllerFactory
+                .brokerEligible(mPublicClientConfiguration.getAppContext(),
+                mPublicClientConfiguration.getDefaultAuthority(),
+                mPublicClientConfiguration)) {
+            //Remove the account from Broker
+            sBackgroundExecutor.submit(new Runnable() {
```
Comment (kreedula) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java]: move the Background operation logic to `BrokerMsalController().removeBrokerAccount` as discussed

**Comment by heidijinxujia** [comment]
@kreedula @iambmelt  Please let me know if there is any other comment. Thanks.

**Comment by heidijinxujia** [comment]
Thank you, Krishna. :)

File: msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java [hunk 2]
```diff
@@ -46,7 +46,7 @@
      * @return A representation of the supplied Account, as an IAccount.
      */
     @NonNull
-    static IAccount adapt(@NonNull final IAccountRecord accountIn) {
+    public static IAccount adapt(@NonNull final IAccountRecord accountIn) {
```
Comment (iambmelt) [inline msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java]: If we're going to make this public, we need to move this class to an `internal` namespace

**Pull request merged**